canonical/canonical.com
# WD-17884 Remote work map pins update

## Done

#1475 did not add Vancouver to the map pins, and duplicated the London pin. This fixes that by adding Vancouver and removing the duplicated London. 

## QA

- Open the remote work page
- See that a pin has been added to the remote map for Vancouver
- The duplicate London pin cannot be QA'd, as it was impossible to tell it was duplicated visually. Code review will suffice .

## Issue / Card

Fixes [WD-17884](https://warthogs.atlassian.net/browse/WD-17884)
